### PR TITLE
Make srpm self contained

### LINKED ISSUE #4: The SRPM is not buildable
~~~
$ create-fake-rpm jre-for-flexibee-rhbz1882240 jre=1.8.0 
Wrote: /home/vondruch/fafa/fake-jre-for-flexibee-rhbz1882240-0-0.src.rpm

$ mock -r fedora-rawhide-x86_64  fake-jre-for-flexibee-rhbz1882240-0-0.src.rpm 

... snip ...

Start: build setup for fake-jre-for-flexibee-rhbz1882240-0-0.src.rpm
warning: line 3: Possible unexpanded macro in: Name: fake-%{fake_name}
warning: line 5: Possible unexpanded macro in: Version: %{fake_version}
warning: line 6: Possible unexpanded macro in: Release: %{fake_release}
error: line 11: Dependency tokens must begin with alpha-numeric, '_' or '/': Provides: %{fake_provides}

... snip ...
~~~

### PR DESCRIPTION
**Pull request by psgreco**
Fixes: https://github.com/xsuchy/create-fake-rpm/issues/4

### CODE REVIEW

File: create-fake-rpm [hunk 1]
```diff
@@ -48,14 +48,22 @@ if [ -z "$NAME" -o -z "$PROVIDES" ]; then
   exit 1
 fi
 
-OUTPUT=$( rpmbuild --define="fake_name $NAME" --define="fake_provides $PROVIDES" \
-        --define="fake_version $VERSION" --define="fake_release $RELEASE" \
+SPECFILE=${PWD}/${NAME}.spec
+>${SPECFILE}
```
Comment (xsuchy) [inline create-fake-rpm]: I would rather prefer here `touch ${SPECFILE}` which is more obvious what we are doing here.

**Comment by xsuchy** [comment]
Otherwise, it is great. Thank you. Please change that one nitpick and I am going to merge this.

**Comment by psgreco** [comment]
Had to add rm -f too (for a possible cleanup)

**Comment by xsuchy** [comment]
Thank you.

**Pull request merged**